test/ui-test/testSuites/suite_messaging/tst_ChatFlow/test.feature

```diff
@@ -69,23 +69,21 @@ Feature: Status Desktop Chat Basic Flows
         When the user clears chat history
         Then the chat is cleared
 
-	@mayfail
-	# TODO: Verification of gif sent fails. And also `tenor GIFs preview is enabled` step doesn't work. Review it.
     Scenario: The user can send a GIF
         Given the user opens app settings screen
         And the user opens the messaging settings
-        And tenor GIFs preview is enabled
-        When the user sends a GIF message
+        When the user tries to activate link preview
```

Comment:
I guess you used `tries` because it might be already activated in another step? If so, maybe something like 
`When the user activates the link preview if it is deactivated`
What do you think?
Otherwise, it sounds like the step may just fail.

Reply:
Done